libdnf/sack/query.cpp

```diff
@@ -1887,17 +1887,43 @@ Query::Impl::filterAdvisory(const Filter & f, Map *m, int keyname)
                 installed_solvables.push_back(pool_id2solvable(pool, installed_id));
             }
             std::sort(installed_solvables.begin(), installed_solvables.end(), NameArchSolvableComparator);
+
+            std::vector<Solvable *> obsolete_solvables;
+            Query obsoletes(sack, ExcludeFlags::IGNORE_EXCLUDES);
+            obsoletes.addFilter(HY_PKG_NEVRA_STRICT, HY_EQ, resultPset);
```

Comment:
I guess that you used an incorrect filter name. May be you wanted to use `HY_PKG`. Is it correct?